src/lib/components/Modal/README.mdx

```diff
@@ -825,21 +826,157 @@ To enable it, you just need to pass a reference to the buttons using
 a reference to the button is that if the button is disabled, the key press will
 not fire the event.
 
-👉 We strongly recommend using this feature together with
-[Autofocus](#autofocus) for a better user experience.
-
-## Autofocus
+## Autofocus And Focus Enforcing
 
 Autofocus is implemented to enhance the user experience by automatically
-focussing an element within the modal.
+focusing an element within the modal.
 
 How does it work? It tries to find `input`, `textarea`, and `select` elements
 inside of Modal and moves focus onto the first non-disabled one. If none is
 found and the `primaryButtonRef` prop on Modal is set, then the primary button
-is focused.
+is focused. If there are neither, it tries to focus any other focusable elements.
+In case there are none, Modal itself is focused.
+
+For most use cases, to make Modal accessible to all users, it should not be possible
+to move focus on elements outside of it.
 
-Autofocus is enabled by default, so if you want to control the focus of
-elements manually, set the `autoFocus` prop on Modal to `false`.
+This behavior is enabled by default and works only if autoFocus is enabled. We strongly
+recommend you keep it this way. If you need to move focus outside of Modal, set
+`focusEnforce` to `true`.
```

Comment:
Should it not say:

> set `focusEnforce` to `true`.

Or am I just confused?

Reply:
Looks like I'm the one who's confused, corrected.